apps/files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -6,17 +6,14 @@
       :selectedRole="role"
       @roleSelected="selectRole"
     />
-    <div v-if="$_ocCollaborators_currentRoleDescription" class="uk-text-muted" v-text="$_ocCollaborators_currentRoleDescription"></div>
-    <hr v-if="$_ocCollaborators_hasAdditionalPermissions" class="divider" />
     <label class="oc-label" v-if="$_ocCollaborators_hasAdditionalPermissions">
-      <translate>Additional permissions</translate>
+      <translate>Additional permissions</translate>:
```

Comment:
keep the colon inside the tag. in some translations the colon could be on the left side I guess (RTL mode)

I googled a bit and found this to confirm: http://docs.translatehouse.org/projects/localization-guide/en/latest/guide/translation/punctuation.html#colons

Also https://www.accelingo.com/software-localization-best-practices/ (search for Punctuation)

Please also adjust the other occurrences

Reply:
That was an interesting read. `For example, there are spaces surrounding a colon in French.` from your last link. 😱 Changed every occurrence of punctuation and i18n that I could find.

Reply:
Yep. It's only recently that I learnt that both German and English do not put a space before a question mark while in French there's one.